app/views/pages/home.html.erb

```diff
@@ -16,6 +16,9 @@
      Copyright: public domain ( modified )
   </div>
 
+  <!-- anchor for featured profiles -->
+  <a id="current_topic"></a>
```

Comment:
wouldn't it be better to call the anchor: featured_profiles? Then we have an consistent naming.

Reply:
I thought, since the anchor is visible in the url, it is better to name it as it looks like for visitors. "Featured profiles" can be understand as privileging some speakerinnen and so bring some rating in it, what we don't want. "Current topic" sounds more related to neutral announcements. But we can obviously change it.

Reply:
You are right, I didn't think about that. 

Should we merge and deploy?

Reply:
If it's actual needed than yes, otherwise I would refactor the featured profiles in this branch: I want to build a many to many relationship (that should enhance the performance/database access) so there will be some name changing.

Reply:
Actually, I would merge the PR and start a new branch for the refactoring.

That way, we can already start using the feature.

Reply:
ok, cool 👍